Wrong key in __odrpack.c

The attribute `Output.work_ind` does not give out all indices correctly, since there is a key mistyped in
https://github.com/scipy/scipy/blob/master/scipy/odr/__odrpack.c#L399

The key for the value `vcv` should read `"vcv"` and not `"sd"`

#### Reproducing code example:

```
import numpy as np
import scipy.odr as spodr
from numpy.testing import assert_array_almost_equal

def func(par, x):
    b0, b1 = par
    return b0 + b1 * x

# generate some data
n_data = 4
x = np.arange(n_data)
y = np.where(x % 2, x+0.1, x-0.1)
x_err = np.full(n_data, 0.1)
y_err = np.full(n_data, 0.1)

# do the fitting
linear_model = spodr.Model(func)
real_data = spodr.RealData(x, y, sx=x_err, sy=y_err)
odr = spodr.ODR(real_data, linear_model, beta0=[0.4, 0.4])
odr.set_job(fit_type=0)
out = odr.run()

sd_ind = out.work_ind['sd']
assert_array_almost_equal(out.sd_beta, out.work[sd_ind:sd_ind+len(out.sd_beta)])
```

#### Error message:

```
AssertionError: 
Arrays are not almost equal to 6 decimals

Mismatched elements: 2 / 2 (100%)
Max absolute difference: 0.09133639
Max relative difference: 10.02956528
 x: array([0.105969, 0.056659])
 y: array([ 0.014633, -0.006275])
```

The work array is nevertheless correct. For a sanity check putting `sd_ind = out.work_ind['sd'] - 2` compares the correct entries as they are enlisted in `vcv` already.

#### Scipy/Numpy/Python version information:
```
1.4.1 1.18.1 sys.version_info(major=3, minor=7, micro=3, releaselevel='final', serial=0)
```

--- a/scipy/odr/__odrpack.c
+++ b/scipy/odr/__odrpack.c
@@ -396,7 +396,7 @@ PyObject *gen_output(F_INT n, F_INT m, F_INT np, F_INT nq, F_INT ldwe, F_INT ld2
             ",s:" F_INT_PYFMT ",s:" F_INT_PYFMT ",s:" F_INT_PYFMT ",s:" F_INT_PYFMT
             ",s:" F_INT_PYFMT ",s:" F_INT_PYFMT ",s:" F_INT_PYFMT ",s:" F_INT_PYFMT
             ",s:" F_INT_PYFMT "}"),
-         "delta", delta, "eps", eps, "xplus", xplus, "fn", fn, "sd", sd, "sd",
+         "delta", delta, "eps", eps, "xplus", xplus, "fn", fn, "sd", sd, "vcv",
          vcv, "rvar", rvar, "wss", wss, "wssde", wssde, "wssep", wssep,
          "rcond", rcond, "eta", eta, "olmav", olmav, "tau", tau, "alpha",
          alpha, "actrs", actrs, "pnorm", pnorm, "rnors", rnors, "prers",

--- a/scipy/odr/tests/test_odr.py
+++ b/scipy/odr/tests/test_odr.py
@@ -472,3 +472,26 @@ class TestODR(object):
         output = odr_obj.run()
         assert_array_almost_equal(output.beta, [1.0, 2.0, 3.0])
 
+    def test_work_ind(self):
+
+        def func(par, x):
+            b0, b1 = par
+            return b0 + b1 * x
+
+        # generate some data
+        n_data = 4
+        x = np.arange(n_data)
+        y = np.where(x % 2, x + 0.1, x - 0.1)
+        x_err = np.full(n_data, 0.1)
+        y_err = np.full(n_data, 0.1)
+
+        # do the fitting
+        linear_model = Model(func)
+        real_data = RealData(x, y, sx=x_err, sy=y_err)
+        odr_obj = ODR(real_data, linear_model, beta0=[0.4, 0.4])
+        odr_obj.set_job(fit_type=0)
+        out = odr_obj.run()
+
+        sd_ind = out.work_ind['sd']
+        assert_array_almost_equal(out.sd_beta,
+                                  out.work[sd_ind:sd_ind + len(out.sd_beta)])

